src/main/java/ee/tuleva/onboarding/auth/mobileid/MobileIDSession.java

```diff
@@ -0,0 +1,56 @@
+package ee.tuleva.onboarding.auth.mobileid;
+
+import ee.sk.mid.MidHashToSign;
+import java.io.Serializable;
+import lombok.Getter;
+
+@Getter
+public class MobileIDSession implements Serializable {
+  private static final long serialVersionUID = -7501351267187058440L;
+
+  private String firstName;
+  private String lastName;
+  private String personalCode;
+  private String challenge;
+  private String sessionId;
+  private String phoneNumber;
+  private MidHashToSign authenticationHash;
+
+  public MobileIDSession(
+      String sessionId, String challenge, MidHashToSign authenticationHash, String phoneNumber) {
+    this.firstName = "";
+    this.lastName = "";
+    this.personalCode = "";
```

Comment:
I think it's better to leave them as `null` instead of initializing to empty strings (I know this class is from the mobile-id library, so not originally written by you, but right now is a good opportunity to improve the design a tiny bit).